Tutorials/Amr/Advection_AmrLevel/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-if ( (DIM EQUAL 1) OR NOT ENABLE_FORTRAN )
+if ( (DIM EQUAL 1) OR NOT CMAKE_Fortran_COMPILER_LOADE )
```

this looks like a typo
```suggestion
if ( (DIM EQUAL 1) OR NOT CMAKE_Fortran_COMPILER_LOADED )
```